src/main/java/com/google/sps/data/LoginStatus.java

```diff
@@ -21,11 +21,13 @@
     private boolean needsToRegister;
     private String url;
     private String userId;
+    private String userEmail;
```

Comment:
I think some of these changes were in your previous two PRs, can you make a new one with just timeslot formatting changes?

Reply:
Now that the commits that this PR depended on are merged only the relevant changes show up. I can manually create a new one though.

Reply:
This works!